course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -985,6 +985,9 @@ def _add_course_instructors(self, course_instructors, course_run):
 
                 PersonSocialNetwork.objects.bulk_update(socials, ['url'])
 
+            instructor.given_name = instructor.given_name.title()
```

Comment:
@iam-mhaseeb @burhandodhy lets send course title and instructor given name in camel case from WordPress instead of converting it in course discovery service ( reducing the number of changes in course discovery service).

Reply:
From slack conversation, we are adding our changes to Wordpress data loader.
https://edlyinc.slack.com/archives/C010MM9C282/p1621854496043700?thread_ts=1621845483.025900&cid=C010MM9C282